bin/assets/scripts/loc.js

```diff
@@ -0,0 +1,2 @@
+const locWidth = document.getElementsByTagName('li').length.toString().length;
+document.documentElement.style.setProperty('--loc-width', `${locWidth - 0.5}em`);
```

Comment:
Could you provide a screenshot of the related bug ?

Reply:
![image](https://user-images.githubusercontent.com/48163546/103156767-9bdf9400-47ac-11eb-9ebf-3feea83c24b9.png)
https://rtdbin.fusetim.tk/yphawa.txt